bzt/modules/apiritif/generator.py

```diff
@@ -586,6 +586,16 @@ def _gen_webdriver(self):
                 targets=[ast.Name(id="options")],
                 value=ast_call(
                     func=ast_attr("webdriver.ChromeOptions"))))
+            body.append(ast.Expr(
+                ast_call(
+                    func=ast_attr("options.add_argument"),
+                    args=[ast.Str("%s" % "--no-sandbox")]
+                )))
+            body.append(ast.Expr(
+                ast_call(
+                    func=ast_attr("options.add_argument"),
+                    args=[ast.Str("%s" % "--disable-dev-shm-usage")]
```

Comment:
is this really necessary? Could you provide the evidence of that for both options?

Reply:
maybe --no-sandbox is enough, i haven't checked it.
people suggested adding them both.
https://stackoverflow.com/questions/50642308/webdriverexception-unknown-error-devtoolsactiveport-file-doesnt-exist-while-t

Reply:
so could you check if it's necessary and let me know?